dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -41,8 +41,6 @@
 # Please, always indicate in comments the feature used for easier bookkeeping.
 
 NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES: List[str] = [
-    # Hardcoded qubit placement
-    'docs/google/qubit-placement.ipynb',
```

Comment:
Does this actually "Enable" the test? Note that `"**/google/*.ipynb"` is part of `SKIP_NOTEBOOKS` below, so my understanding was that all google/* notebooks are not tested by default.

Reply:
Ack, it doesn't seem to launch it off, but might still be worth removing since hardcoded placement is now public. WDYT ?